src/tests/components/SearchForm.test.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import SearchForm from "../../components/SearchForm";
+
+describe("Search Form feature", () => {
+  const validProps = {
+    searchText: "Liverpool",
+    setSearchText: () => {},
+    onSubmit: () => {},
+  };
+
+  it("Snapshot test", () => {
+    const { asFragment } = render(
+      <SearchForm
+        searchText={validProps.searchText}
+        setSearchText={validProps.setSearchText}
+        onSubmit={validProps.handleCitySearch}
```

Comment:
Both the render methods should change this line so it matches the actual valid props:
```
        onSubmit={validProps.onSubmit}
```